src/components/Personal-Zone/Publisher/JobsAppEmployerBody.tsx

```diff
@@ -21,27 +21,27 @@ const styles = makeStyles({
 });
 
 interface JobsAppEmployerBodyOwnProps {
-    jobsStore?: JobsStore;
+    userStore?: UserStore;
 }
 
 const JobsAppEmployerBody: React.FC<JobsAppEmployerBodyOwnProps> = (props): JSX.Element => {
     const classes = styles({});
-    const jobsStore: JobsStore = props.jobsStore!;
+    const userStore: UserStore = props.userStore!;
 
     useEffect(() => {
-        jobsStore.loadAdvertisements();
-    }, [jobsStore]);
+        userStore.loadRamadAds();
+    }, [userStore]);
 
-    const showEmptyDisplay = false && jobsStore.advertisements.length > 0; 
+    const showEmptyDisplay = userStore.getRamadAds.length === 0

```

Comment:
💯

Reply:
add semicolon

Reply:
that is a js file not ts, added semicolon